spec/features/write_to_councillors_spec.rb

```diff
@@ -16,13 +16,16 @@
     end
 
     scenario "can’t see councillor messages sections" do
-      page.should_not have_content("Write to your elected councillors about this application")
+      page.should_not have_content("Who should this go to?")
       # TODO: and you should not be able to write and submit a message.
     end
   end
 
   context "when logged in as admin" do
     background do
+      @council_or_councillor_explanation = "Write to the council
```

Comment:
* This isn't really part of setting up the test so it should be in-lined.
* I find it confusing splitting up this line of text ("why is this line split like this? is it supposed to be telling me something?"). Furthermore we don't really need to read the text to understand the code - it's just a string we're testing for on the page.
* Instead of setting an instance variable you should [use `let`](http://betterspecs.org/#let). It's aliased to `given` in Capybara too.